rpcs3/evdev_joystick_handler.cpp

```diff
@@ -168,6 +168,30 @@ void evdev_joystick_handler::Close()
     }
 }
 
+inline int Deadzone(f32 input)
+{
+	int	deadzone=g_evdev_joystick_config.deadzone;
+	float deadzone_a=128.0/(128.0-float(deadzone));
+	float deadzone_b=255.0*(1.0-deadzone_a);
+	if (input>=128-deadzone&&input<=128+deadzone)
+	{
+		input=128;
+	}
+	else
+	{
+		if (input>128+deadzone)
+		{
+			input=deadzone_a*(float)input+deadzone_b;
+		}
+		if (input<128-deadzone)
+		{
+			input=(128.0/(128.0-(float)deadzone))*input;
+		}
+	}
+    return static_cast<int>(input);
```

Comment:
Tabs or spaces, sure, but please don't mix both...the entire file is using spaces, and this PR is using tabs except for one line, which is also using spaces... :confused: